src/constants/imgPaths.ts

```diff
@@ -0,0 +1,121 @@
+import androidStudio128 from "../imgs/editor-icons/android-studio-128.png";
```

Comment:
Let's add a comment top of this file indicating it's a generated file, with the yarn script command that generates it.